<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>541</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Gestion</string>
     </attribute>
     <widget class="QTableView" name="tab_matchs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>20</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>discipline</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id_equipe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id_match</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date du match</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ajouter">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="la_daten">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_matricule">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLineEdit" name="la_daten_modif">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom_modif">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id_equipe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_matricule_modif">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_modif">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>discipline</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date du match</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id_match</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_modifier">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ancien id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_matricule_ancien_modif">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="le_matricule_sup">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_supprimer">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id_match</string>
        </property>
       </widget>
      </widget>
     </widget>
     <zorder>tabWidget_2</zorder>
     <zorder>tab_matchs</zorder>
     <zorder>line_3</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>case sensitive</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>liste</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>16</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>par colonne</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>mot clé</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>381</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>411</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Résultats</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>16</width>
        <height>191</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox</zorder>
     <zorder>pushButton</zorder>
     <zorder>checkBox</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>comboBox</zorder>
     <zorder>listView</zorder>
     <zorder>verticalScrollBar_4</zorder>
     <zorder>line</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Saisie Resultat</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>liste originale</string>
      </property>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>16</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>liste triée</string>
      </property>
      <widget class="QTableView" name="tableView_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>16</width>
         <height>141</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>par colonne</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>par date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Affichage matchs</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>256</width>
        <height>190</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>liste</string>
      </property>
      <widget class="QTableView" name="tableView_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>16</width>
         <height>181</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>go to date</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Matchs clés</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>liste</string>
      </property>
      <widget class="QTableView" name="tableView_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_6">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>16</width>
         <height>181</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>381</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Reporter/Alerte</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>liste</string>
      </property>
      <widget class="QTableView" name="tableView_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar_7">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>16</width>
         <height>141</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>341</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Objet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Texte</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
